frontend/src/layouts/Dashboard.tsx

```diff
@@ -43,6 +47,10 @@ function Dashboard() {
                     <Route path={'/purchases/update/:id'} element={<PurchaseUpdate/>}/>
                     <Route path={'/purchases/detail/:id'} element={<PurchaseDetail/>}/>
                     <Route path={'/purchases/add'} element={<PurchaseAdd/>}/>
+                    <Route path={'/orders'} element={<OrderList/>}/>
+                    <Route path={'/orders/detail/:id'} element={<OrderDetails/>}/>
+                    <Route path={'/orders/add'} element={<OrderAdd/>}/>
+                    <Route path={'/order-details/add/:orderId'} element={<OrderDetailAdd/>}/>
```

Comment:
here the same :)

Reply:
> here the same :)

Element ist kein Link. Das ist Component Name. orderId ist Variablenname